sched/Kconfig

```diff
@@ -318,6 +318,12 @@ config TICKET_SPINLOCK
 
 endif # SPINLOCK
 
+config RW_SPINLOCK
+	bool "Support read-write Spinlocks"
+	default y
+	---help---
+		Support Read-write spinlock
```

Comment:
Add complete help with the 'Why questions answered.

Reply:
Done.